### PR TITLE
Fixes negative bodytemp in some cases and shivering not doing anything unless you're already very cold

### PR DESCRIPTION
**Pull request by DaedalusGame**
:cl:
fix: shivering symptom now works properly instead of only cooling you if you're already cold
fix: fixed bodytemp going negative in a few cases
/:cl:

hello you can use `adjust_bodytemperature(-n)` next time

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
I'd almost suggest an AdjustBodyTemp() proc that clamps it at 0 rather than having to always repeat the max on literally every cooling mechanic.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Do what cobby suggested

**Comment by DaedalusGame** [comment]
this kills the man

**Comment by ExcessiveUseOfCobblestone** [comment]
Not by negative temperatures though :^)

**Comment by DaedalusGame** [comment]
well it needs to be a two-parameter proc and it needs to replace every clamp we already have, which is gonna be annoying

**Comment by ExcessiveUseOfCobblestone** [comment]
what do you mean by that...

**Comment by DaedalusGame** [comment]
ha that's what you think but some temperature changes don't want to go lower than some arbitrary temperature, like T0C

**Comment by ExcessiveUseOfCobblestone** [comment]
lame and gay and probably should remove them :^)

**Comment by DaedalusGame** [comment]
why does dunc own code in every single pr i touch

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -112,7 +112,7 @@
 
 	var/loc_temp = get_temperature(environment)
 
-	bodytemperature += adjust_body_temperature(bodytemperature, loc_temp, 1)
+	adjust_bodytemperature(adjust_body_temperature(bodytemperature, loc_temp, 1))
```
Comment (DaedalusGame) [inline code/modules/mob/living/simple_animal/slime/life.dm]: this is the future you chose

**Comment by ExcessiveUseOfCobblestone** [comment]
It's just one spot it'll be ok someone will make it prettier later :)

File: code/modules/mob/living/simple_animal/slime/life.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/slime/life.dm]: It's just one spot it'll be ok someone will make it prettier later :)

**Pull request merged**